app/web/features/profile/locales/en.json

```diff
@@ -127,37 +128,68 @@
+      "message_surfed": "You <strong>will not be able to write a reference</strong> if you indicate you didn't stay with {{name}}. They will not be notified but they can still leave a reference for you.<br /><br />If you need to report something find out how <2>here</2>.",
+      "confirm_hosted": "Confirm I didn't host",
+      "confirm_surfed": "Confirm I didn't surf"
+    },
+    "did_stay_explanation": "Select 'No' if you didn't actually meet. You will not be able to write a public reference, but can optionally share <strong>private</strong> feedback with us about what happened. The other person won’t be informed, but can still leave a reference if they say you met.",
     "host_request_reference_explanation": "Host request references are only made visible once both references have been written, or after 2 weeks. Hold tight!",
     "invalid_reference_type": "Invalid reference type",
     "invalid_step": "Invalid form step",
     "previous_step": "Previous step",
-    "private_answer": "Your answer will not be seen by the other person, and will remain private.",
+    "private_answer": "Your answers on this page are private, and will only be seen by the Couchers.org Safety Team.",
     "private_text_label": "You will also submit the following private answers:",
     "public_answer": "This will appear publicly in the References section of their profile.",
-    "public_text_label": "You are leaving the following reference on your fellow Coucher's Guestbook:",
+    "public_text_label": "You are leaving the following public reference which will appear on {{name}}'s profile:",
     "rating_label": "Your experience made you feel like ",
     "rating_explanation": "- **Negative:** You did not enjoy this person's company.\n- **Neutral:** You didn't have strong feelings either way about this person. Their community standing will not be affected.\n- **Positive:** You had a wonderful time with this person. This will improve their community standing.\n- **Amazing:** This person exceeded all your expectations and is an asset to the Couchers community.",
-    "rating_how": "How should I rate my experience?",
+    "rating_how": "How was your overall experience with {{name}}?",
     "rating_question": "How would you rate your overall experience with {{name}}?",
     "reference_form_heading_friend": "You met with {{name}}",
     "reference_form_heading_hosted": "You hosted {{name}}",
     "reference_form_heading_surfed": "You surfed with {{name}}",
+    "reference_form_heading_did_stay_surfed": "Did you stay with {{name}}?",
+    "reference_form_heading_did_stay_hosted": "Did you host {{name}}?",
     "writing_for_text": "You are writing a reference for the following person:",
-    "reference_submit_heading": "Thank you for leaving a reference!",
+    "reason_didnt_meetup": "Why didn't you meet up? (Optional)",
+    "reference_submit_heading": "Please review your reference",
     "reference_success": "Successfully wrote the reference!",
```

Comment:
This is a bit ambiguous. We should either say "You successfully..." OR "Reference successfully submitted!" 

```suggestion
    "reference_success": "Reference successfully submitted!",
```

Reply:
This is on the dialog to submit the reference, so at this point it's not successfully submitted yet.

Is there something else you want it to say given that?

Reply:
Okay looked into this. I decided to remove this dialog. I added a thank you page it redirects to instead via the Figma design. It looks like the redirect was broken though, but now it works! The thank you page seems more professional thank a pop-up dialog anyway, and makes the dialog redundant.

I just host requested you now so if you accept you should be able to test it again, but this is what it looks like:

<img width="1084" alt="Screenshot 2025-05-20 at 8 42 28 PM" src="https://github.com/user-attachments/assets/478caa74-7755-4f5c-9fc2-43b161bf5f37" />

Reply:
It should include the 2 weeks text though for host/surf requests though.

Reply:
>It should include the 2 weeks text though for host/surf requests though.

I totally agree. Replace the "Did you have a good experience" with the text "References become visible when both people have submitted their references, or after 2 weeks, whichever comes sooner."